examples/stable-diffusion/training/README.md

```diff
@@ -163,86 +163,95 @@ Install the requirements:
+  --use_hpu_graphs_for_training \
+  --use_hpu_graphs_for_inference \
+  --validation_prompt="a robotic cat with wings" \
+  --validation_epochs 48 \
+  --checkpointing_steps 336 \
+  --mediapipe dataset_sdxl_pokemon \
+  --adjust_throughput
+```
 
+### Single-card Training on Gaudi1
 ```bash
-python train_text_to_image_sdxl.py \
-  --pretrained_model_name_or_path="stabilityai/stable-diffusion-xl-base-1.0" \
-  --pretrained_vae_model_name_or_path="madebyollin/sdxl-vae-fp16-fix" \
-  --dataset_name="lambdalabs/pokemon-blip-captions" \
-  --caption_column="text" \
-  --resolution=1024 --random_flip \
-  --train_batch_size=1 \
-  --num_train_epochs=2 --checkpointing_steps=500 \
-  --learning_rate=1e-04 --lr_scheduler="constant" --lr_warmup_steps=0 \
-  --seed=42 \
-  --output_dir="sd-pokemon-model-lora-sdxl" \
-  --finetuning_method="lora" \
-  --gaudi_config_name="Habana/stable-diffusion" \
-  --throughput_warmup_steps=3 \
-  --use_hpu_graphs \
+PT_HPU_MAX_COMPOUND_OP_SIZE=5 python train_text_to_image_sdxl.py \
+  --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 \
+  --pretrained_vae_model_name_or_path stabilityai/sdxl-vae \
+  --dataset_name lambdalabs/pokemon-blip-captions \
+  --resolution 512 \
+  --center_crop \
+  --random_flip \
+  --proportion_empty_prompts=0.2 \
+  --train_batch_size 1 \
+  --gradient_accumulation_steps 4 \
+  --max_train_steps 3000 \
+  --learning_rate 1e-05 \
+  --max_grad_norm 1 \
+  --lr_scheduler constant \
+  --lr_warmup_steps 0 \
+  --output_dir sdxl-pokemon-model \
+  --gaudi_config_name Habana/stable-diffusion \
+  --throughput_warmup_steps 3 \
+  --use_hpu_graphs_for_training \
+  --use_hpu_graphs_for_inference \
   --bf16
 ```
 
-> [!NOTE]
-> SDXL's VAE is known to suffer from numerical instability issues. This is why we also expose a CLI argument namely `--pretrained_vae_model_name_or_path` that lets you specify the location of a better VAE (such as this one).
-
-#### LoRA SDXL Inference
-
-Once you have trained a LoRA weights as in the example above, inference can be done
-by using the `GaudiStableDiffusionXLPipeline`.
-
-```python
-import torch
-from optimum.habana.diffusers import (
-    GaudiStableDiffusionXLPipeline,
-    GaudiEulerDiscreteScheduler,
-)
-
-model_id = "stabilityai/stable-diffusion-xl-base-1.0"
-lora_model_id = "sd-pokemon-model-lora-sdxl"
-pipe = GaudiStableDiffusionXLPipeline.from_pretrained(
-  model_id,
-  scheduler=GaudiEulerDiscreteScheduler.from_pretrained(model_id, subfolder="scheduler"),
-  torch_dtype=torch.bfloat16,
-  use_habana=True,
-  use_hpu_graphs=True,
-  gaudi_config="Habana/stable-diffusion",
-)
-pipe.load_lora_weights(lora_model_id)
-
-prompt = "cute dragon creature"
-image = pipe(prompt).images[0]
-image.save("green-pokemon.png")
-```
+> **_NOTE:_** There is a known issue that in the first 2 steps, graph compilation takes longer than 10 seconds. This will be fixed in a future release.
+> **_NOTE:_** `--mediapipe` only works on Gaudi2.
```

Comment:
```suggestion
> [!NOTE]
> There is a known issue that in the first 2 steps, graph compilation takes longer than 10 seconds. This will be fixed in a future release.

> [!NOTE]
> `--mediapipe` only works on Gaudi2.
```
This renders better